utils/src/test/scala/za/co/absa/enceladus/utils/broadcast/BroadcastUtilsSuite.scala

```diff
@@ -18,8 +18,9 @@ package za.co.absa.enceladus.utils.broadcast
 import org.apache.spark.sql.functions._
 import org.apache.spark.sql.{DataFrame, Row}
 import org.scalatest.wordspec.AnyWordSpec
-import za.co.absa.enceladus.utils.error.Mapping
+import za.co.absa.enceladus.utils.error.EnceladusErrorMessage
```

Comment:
this is not used

Reply:
I don't have that file in my local branch.

Reply:
how's that possible, did you remove it? What `git status` tells you?